build/src/src/modules/sshPassword.js

```diff
@@ -0,0 +1,86 @@
+const shell = require("utils/shell");
+
+const insecureSalt = "insecur3";
+
+/**
+ * The password is set in the pre-seed, only affects ISO-dappnodes
+ * https://github.com/dappnode/DAppNode_Installer/blob/472d046f2661d737c4650a1f895c8c4d89234cd8/build/dappnode/scripts/preseed_unattended.cfg#L12
+ * `d-i passwd/user-password-crypted password $6$insecur3$rnEv9Amdjn3ctXxPYOlzj/cwvLT43GjWzkPECIHNqd8Vvza5bMG8QqMwEIBKYqnj609D.4ngi4qlmt29dLE.71`
+ *
+ * The file `/etc/shadow` contains:
+ * - with the original salt and password
+ * `dappnode:$6$insecur3$rnEv9Amdjn3ctXxPYOlzj/cwvLT43GjWzkPECIHNqd8Vvza5bMG8QqMwEIBKYqnj609D.4ngi4qlmt29dLE.71:18004:0:99999:7:::`
```

Comment:
I think we are giving too much detail here, that can be exploited, so I would remove all those comments

Reply:
Ok, deleted comments